python/graphscope/nx/tests/test_nx.py

```diff
@@ -356,11 +369,6 @@ def test_error_on_wrong_nx_type(self):
         with pytest.raises(TypeError):
             nx_g = nx.DiGraph(g)
 
-    def test_error_on_duplicate_oid(self):
-        g = self.duplicated_oid_g
-        with pytest.raises(AnalyticalEngineInternalError):
-            nx_g = self.NXGraph(g)
```

Comment:
Why we drop this test case?

Reply:
since we add label support, there would be no duplicated oid.